examples/apps/aml/web-client/README.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+-->
+
+# Santander AML DEMO SGX WEB CLIENT
+
+## Compiles and hot-reloads for development
+```
+npm install
+npm run serve
+```
+
+## Deployment in production
+
+
+1. Go to **Project Root Folder**
+
+2. Build Trusted Compute Framework using "docker-compose-aml-web-client.yaml"
+
+```
+docker-compose -f docker-compose-aml-web-client.yaml up
+```
+
+3. Go back to to this folder (web-client)
+
+4. Change variable "SERVER_URL" in file "src/config.js"
```

Comment:
What is value of SERVER_URL is set to?

Reply:
SERVER_URL : 'http://{YOUR DOMAIN}.com/',